static/js/tinderAMatch.js

```diff
@@ -0,0 +1,84 @@
+let readingList = [];
+let bannedList = [];
+
+async function setCurrentArticles() {
+    await fetch('/tinder/articleList')
+    .then(function(res) {
+        return res.text()
+    })
+    .then(function(res) {
+        filteredArticleList = filterArticles(res)
+        sessionStorage.setItem('currentArticles', filteredArticleList)
+    })
+    setCurrentDate();
+}
+
+function filterArticles(articleList) {
+    articleList = JSON.parse(articleList)
+    
+    // TODO: filter out banned + reading lists from the article list before saving
+    
+    return JSON.stringify(articleList)
+}
+
+function addListeners() {
+    document.getElementById('superlike').addEventListener('click', function() {
+        window.open(getFirstCurrentArticle().link);
+        addToReadingList();
+        setCurrentDate();
+    })
+    document.getElementById('like').addEventListener('click', function() {
+        addToReadingList();
+    });
+    document.getElementById('dislike').addEventListener('click', function() {
+        addToBannedList();
+    });
+    document.getElementById('readingList').addEventListener('click', function() {
+        window.open('/tinder/myType')
+    });
+}
+
+function getReadingAndBannedLists() {
+    localReadingList = localStorage.getItem('readingList')
+    localBannedList = localStorage.getItem('bannedList')
+    if(localReadingList) readingList = JSON.parse(localReadingList)
+    if(localBannedList) bannedList = JSON.parse(localBannedList)
+}
+
+function getCurrentArticles() {
+    return JSON.parse(sessionStorage.getItem('currentArticles'))
+}
+
+function getFirstCurrentArticle() {
+    return getCurrentArticles()[0]
+}
+
+function addToReadingList() {
```

Comment:
addToReadingList and addToBanned list do essentially the same thing. I'd merge them into one function and use an argument to toggle between the 2.